DeutschJozsaAlgorithm/Tests.qs

```diff
@@ -111,9 +107,9 @@ namespace Quantum.Kata.DeutschJozsaAlgorithm {
         
         // the mask with only the K-th element set to 1 corresponds to Oracle_Kth_Qubit
         for (i in 0 .. L - 1) {
-            set r[i] = 1;
+            set r w/= i <- 1;
```

Comment:
I think this `set` statement and the one at L112 can both be removed entirely in favor of replacing `r` in the next line by `r w/ i <- 1`.

Reply:
You are right of course - missed that. Done.